### PR TITLE
Add external links and internal routes

### PR DESCRIPTION
**Pull request by akanuri9**
https://oncprojectracking.healthit.gov/support/browse/SITE-3877

### CODE REVIEW

File: src/components/shared/nav/nav/categories/SiteNavIndustryTools.tsx [hunk 1]
```diff
@@ -23,20 +22,20 @@ export default function SiteNavIndustryTools() {
...
+      isExternalLink: false,
+      href: '/hl7-tools',
       icon: <MonitorHeartOutlined />,
     },
     {
       text: 'Reference Data',
-      isExternalLink: true,
-      href: GENERIC_LINK_TO_REPLACE,
+      isExternalLink: false,
+      href: 'reference-data',
```
Comment (drbgfc) [inline src/components/shared/nav/nav/categories/SiteNavIndustryTools.tsx:32]: missing slash

File: src/components/shared/nav/nav/categories/SiteNavIndustryTools.tsx [hunk 2]
```diff
@@ -23,20 +22,20 @@ export default function SiteNavIndustryTools() {
...
       icon: <FolderSpecialOutlined />,
     },
     {
       text: 'Implementation Guide Authoring Tools',
-      isExternalLink: true,
-      href: GENERIC_LINK_TO_REPLACE,
+      isExternalLink: false,
+      href: 'implementation-guide-authoring-tools',
```
Comment (drbgfc) [inline src/components/shared/nav/nav/categories/SiteNavIndustryTools.tsx:38]: needs slash prefix

File: public/alternative-test-methods/tempFileForStructure-deleteWhenFilesAdded
Comment (drbgfc) [inline public/alternative-test-methods/tempFileForStructure-deleteWhenFilesAdded:1]: Thanks for adding these!

**Review by drbgfc — APPROVED** [review]
Looks good, thanks. Added the slashes, merging

**Comment by akanuri9** [comment]
Thank you!

**Pull request merged**